SECURITY-RELEASE-PROCESS.md

```diff
@@ -0,0 +1,188 @@
+# Security Release Process
+
+CoreDNS project has adopted this security disclosures and response policy
```

Comment:
The CoreDNS project ... to ensure responsible handling of ...

Reply:
ok